Bottom: 00c262722c96ee13a9b83c4516933e9416c94e48
Top:    31085ead33a6bfe2a35fe6f724b8d359d6e7ddb2
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:16:16 +0000

DEBUG: Polygon parentage on


---

diff --git a/src/polygon1.c b/src/polygon1.c
index 8041c05..efece45 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2964,13 +2964,14 @@ poly_Boolean_free (POLYAREA * ai, POLYAREA * bi, POLYAREA ** res, int action)
 
   *res = NULL;
 
-#if 0
+#if 1
   /* Make copies for tracking polygon parentage (DEBUG) */
   if (!poly_M_Copy0 (&a_copy, a) || !poly_M_Copy0 (&b_copy, b))
       return err_no_memory;
-#endif
+#else
   a_copy = NULL;
   b_copy = NULL;
+#endif
 
   /* Move the parentage information over onto the copy */
   if (a_copy != NULL)
